kernel/extism-context.wat

```diff
@@ -0,0 +1,3 @@
+(module
+  (global (export "extism_context") (mut externref) (ref.null extern))
```

Comment:
We could add a layer of protection around this by saying that "we don't accept any Wasm module that imports `extism_context`." (This would protect against a malicious plugin that imports this externref, holds onto it across two calls, replacing the global value of the second call with the value from the first call.)

Reply:
Yeah my first thought is we don't want people to be able to poke this and use some other plugin-runner's context

Reply:
returning an error if a plugin is trying to import the global directly sounds like a good idea - I also noticed that the context global gets reset before each call regardless of whether `call_with_context` is used, which makes me confident that a plugin couldn't access the context from another call.